packages/core/src/interpreter.ts

```diff
@@ -287,6 +290,9 @@ export class Interpreter<
     }
 
     this.system._register(this.sessionId, this);
+    if (this._systemId) {
+      this.system._set(this._systemId, this);
+    }
```

Comment:
This happens in `.start()` now. I didn't remove the call from the constructor though. it creates a little bit of a weird effect but I'm not sure what we can do about it:
```ts
const root = interpret(m)
// actors in the system are available here
root.start()
// actors in the system are still available
root.stop()
// actors in the system are *not* available here
root.start()
// actors in the system are available again
```

I don't think this is a big issue but it's likely that our users will prove me wrong. Maybe we could consider a hybrid approach - similar to the one that we mentioned yesterday. Registered actors could always be cleared when they are stopped *except* for the top-level manual `.stop()` on the root.

Reply:
Let's leave it as-is for now; can always address if it comes up.